cockroachcloud/monitoring-page.md

```diff
@@ -7,12 +7,12 @@ docs_area: manage
 
 The **Monitoring** page is accessible on {{ site.data.products.dedicated }} clusters. This page allows you to:
 
-- Set up cluster [monitoring with Datadog](#monitor-with-datadog).
+- Set up cluster [monitoring with Datadog](#monitor-cockroachdb-dedicated-with-datadog).
 - Access the cluster's [built-in DB Console](#access-the-db-console) to view time-series data on SQL queries, troubleshoot query performance, view a list of jobs, and more.
 
 To view the **Monitoring** page, [log in](https://cockroachlabs.cloud/) and click **Monitoring** on the left-hand navigation.
 
-## Monitor with Datadog
+## Monitor {{ site.data.products.dedicated }} with Datadog
```

Comment:
This is not rendering properly https://www.cockroachlabs.com/docs/cockroachcloud/monitoring-page
<img width="472" alt="Screen Shot 2022-10-18 at 10 43 35 AM" src="https://user-images.githubusercontent.com/6822430/196505595-72a3ab37-8166-41ca-abe6-b11b72572f57.png">